src/main/java/org/jvnet/hudson/plugins/collapsingconsolesections/CollapsingSectionNote.java

```diff
@@ -42,13 +42,23 @@
     private String sectionStartPattern;
     private String sectionEndPattern;
     private boolean collapseOnlyOneLevel;
-    
+    private boolean collapseSection;
+
     @DataBoundConstructor
+    public CollapsingSectionNote(String sectionDisplayName, String sectionStartPattern, String sectionEndPattern, boolean collapseOnlyOneLevel, boolean collapseSection) {
+        this.sectionDisplayName = sectionDisplayName;
+        this.sectionStartPattern = sectionStartPattern;
+        this.sectionEndPattern = sectionEndPattern;
+        this.collapseOnlyOneLevel = collapseOnlyOneLevel;
+        this.collapseSection = collapseSection;
+    }
+
     public CollapsingSectionNote(String sectionDisplayName, String sectionStartPattern, String sectionEndPattern, boolean collapseOnlyOneLevel) {
         this.sectionDisplayName = sectionDisplayName;
         this.sectionStartPattern = sectionStartPattern;
         this.sectionEndPattern = sectionEndPattern;
         this.collapseOnlyOneLevel = collapseOnlyOneLevel;
+        this.collapseSection = false;
```

Comment:
call the `this(sectionDisplayName, sectionStartPattern, ...., false)` constructor